src/message/displaywindowComponent.js

```diff
@@ -2,6 +2,7 @@ goog.provide('ngeo.message.displaywindowComponent');
 
 goog.require('ngeo'); // nowebpack
 // webpack: import 'jquery-ui/ui/widgets/resizable.js';
+// webpack: import 'jquery-ui/ui/widgets/draggable.js';
```

Comment:
@gberaudo It is ok to import both lib here ? Because (for mobile) this component can disable drag and resize function. So in some case, we don't want to have it. So perhaps we should have these import here, perhaps it's better 
 to load them in the final app. What's your point of view ?

Reply:
For the record, since the impact is small we decided to keep the dependency here for ease of use.